charts/trusted-issuers-registry/templates/configmap.yaml

```diff
@@ -30,4 +30,26 @@ data:
     {{- end }}
     {{- with .Values.tir.additionalConfigs }}
     {{- toYaml . | nindent 4 }}
-    {{- end }}
\ No newline at end of file
+    {{- end }}
+---
+{{- if .Values.ccs.endpoint -}}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "tir.fullname" . }}-ccs
+  labels:
+    {{- include "tir.labels" . | nindent 4 }}
+data:
+  init.sh: |-
+    curl -X 'POST' \
+      '{{ .Values.ccs.endpoint }}service' \
+      -H 'accept: */*' \
+      -H 'Content-Type: application/json' \
+      -d '{
+      "id": "{{ .Values.ccs.id }}",
+      "default_scope": "{{ .Values.ccs.defaultScope }}",
+      {{- if .Values.ccs.scopes -}}
+      "scopes": {{- toJson .Values.ccs.scopes }}
```

Comment:
Is the name of the field not oidcScopes? https://github.com/FIWARE/credentials-config-service/blob/main/api/credentials-config-service.yaml#L234

Reply:
You are right, this was not up-to-date in the ticket.
Fixed!